.github/workflows/release.yml

```diff
@@ -1,49 +1,28 @@
----
-name: Gem Release
+name: 'Release'
```

Comment:
I'm not sure if the changes in this file are correct. The release.yml in main creates the actual gem release and publishes it, https://github.com/OpenVoxProject/shared-actions/blob/main/.github/workflows/release.yml increments the version? And I'm not sure if that even works for here.

Reply:
ahhhh so one workflow to provide the changelog, one to create the tag and the last one creates the rubygems.org release.